### PR TITLE
Add support for dropdown border variables

### PR DESCRIPTION
**Pull request by vicocamacho**
Add the Bootstrap variables for the arrow of the datepicker:

screen shot 2015-01-14 at 9 30 30 pm

### CODE REVIEW

File: build/build_standalone3.less [hunk 1]
```diff
@@ -21,6 +21,8 @@
 @component-active-bg:           @brand-primary;
 @dropdown-link-active-bg:       @component-active-bg;
 @dropdown-link-hover-bg:        @dropdown-link-active-bg;
+@dropdown-fallback-border:      @dropdown-fallback-border;
+@dropdown-bg:                   @dropdown-bg;
```
Comment (korun) [inline build/build_standalone3.less]: Why we need these lines? O_o

Comment (vicocamacho) [inline build/build_standalone3.less]: You are right, these are not necessary i removed them and now i am using the correct variable for the arrow border (`@dropdown-border`) https://github.com/eternicode/bootstrap-datepicker/commit/388f192ebde778837484c1521b91a087a04182f1

**Comment by vsn4ik** [comment]
:+1: for merge.

**Comment by acrobat** [comment]
@vicocamacho can you add the variable also to the tbws2 less file? and add the parameter to the build.less & build3.less file with the color code currently in the datepicker(3).less file

**Comment by vicocamacho** [comment]
Sure, will do it in the next few days

**Comment by vicocamacho** [comment]
I don't know if you guys saw this, is this what was needed? thank you!

**Comment by acrobat** [comment]
Just add the variables to the `build.less` and `build_standalone.less`. Then this PR should be ready (will review it fully then!)

**Comment by vicocamacho** [comment]
is this correct? https://github.com/vicocamacho/bootstrap-datepicker/commit/6faf3b4b560988bdabc30ba198049a49608a3ecc

**Comment by hebbet** [comment]
will look into it in the next few weeks

**Comment by hebbet** [comment]
LGTM, thanks, merged.

**Comment by acrobat** [comment]
Maybe we should add a styling section to the docs, where we can describe this feature

**Pull request merged**